src/development/LocalTest/Controllers/HomeController.cs

```diff
@@ -218,9 +221,22 @@ public ActionResult FrontendVersion()
                         Text = "localhost:8080 (local dev)",
                         Value = "http://localhost:8080/"
                     }
-                    // TODO: list all old versions from CDN
                 }
             };
+            var cdnVersionsString = await client.GetStringAsync("https://altinncdn.no/toolkits/altinn-app-frontend/index.json");
+            var groupCdnVersions = new SelectListGroup() { Name = "Older versions from cdn" };
```

Comment:
Not all of them are 'older', technically. :wink: 
```suggestion
            var groupCdnVersions = new SelectListGroup() { Name = "Specific version from cdn" };
```

Looks good! Looking forward to test this out.